src/snapshotcloneserver/clone/clone_service_manager.cpp

```diff
@@ -231,6 +232,25 @@ int CloneServiceManager::Flatten(
 
     auto cloneInfoMetric = std::make_shared<CloneInfoMetric>(taskId);
     auto closure = std::make_shared<CloneClosure>();
+
+    // init dlock
+    if (!useMockDLock_) {
```

Comment:
unit test like this test nothing, add some intergration test

Reply:
use gmock in test

Reply:
alread use MockDLock, but the dlock is inited in Flatten based on cloneInfo.destId
even if set the mockDlock, it will be overwritten here. so a judgment is needed here anymore.

have modified the code.